src/test/java/uk/gov/hmcts/divorce/common/event/SubmitConditionalOrderTest.java

```diff
@@ -59,4 +65,23 @@ void shouldSetDateSubmittedOnAboutToSubmit() {
 
         assertThat(response.getData().getConditionalOrder().getDateSubmitted()).isEqualTo(getExpectedLocalDateTime());
     }
+
+    @Test
+    void shouldSendEmailOnAboutToSubmitIfApplicantIsNotRepresented() {
+
+        setMockClock(clock);
+
+        final CaseData caseData = CaseData.builder()
+            .applicant1(Applicant.builder().build())
+            .conditionalOrder(ConditionalOrder.builder().build())
+            .build();
+
+        final CaseDetails<CaseData, State> caseDetails = new CaseDetails<>();
+        caseDetails.setId(1L);
+        caseDetails.setData(caseData);
+
+        final AboutToStartOrSubmitResponse<CaseData, State> response = submitConditionalOrder.aboutToSubmit(caseDetails, null);
```

Comment:
is there a need for the response variable since it's not used

Reply:
Have actioned, well spotted. Cheers!